components/ProjectCard/index.js

```diff
@@ -0,0 +1,35 @@
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function ProjectCard({ project }) {
```

Comment:
```suggestion
export default function ProjectCard({ project }) {
const { imageUrl, title, complexity } = project;
```

I would recommend destructuring here, but only because I think it prettier 🤓
Just a recommendation, you don't have to do this.